flang/test/Driver/lit.local.cfg

```
@@ -0,0 +1,10 @@
+import re
+from lit.llvm import llvm_config
+
+# Drop -isysroot flag for darwin-version.f90.
+idx, driver = next(((i, subst)
+    for i, subst in enumerate(config.substitutions) if "flang" in subst[0]))
```

```suggestion
driver = next((subst
    for subst in config.substitutions if "flang" in subst[0]), None)
```
`i` is not used; don't fail on `StopIteration` if it is not in the list.